include/rosdiscover-clang/Value/Value.h

```diff
@@ -28,6 +28,10 @@ class SymbolicValue : public SymbolicExpr {
   static SymbolicValueType getSymbolicType(clang::QualType clangType) {
     clangType = clangType.getUnqualifiedType();
     auto typeName = clangType.getAsString();
+    return getSymbolicType(typeName);
+  }
+
+  static SymbolicValueType getSymbolicType(std::string typeName) {
```

Comment:
very minor: prefer `std::string const &typeName` to avoid doing an unnecessary copy